mi/dataset/parser/metbk_a_dcl.py

```diff
@@ -140,68 +153,34 @@ def parse_file(self):
-        sea_surface_temperature         = raw_data[6]
-        sea_surface_conductivity        = raw_data[7]
-        shortwave_irradiance            = raw_data[8]
-        eastward_wind_velocity          = raw_data[9]
-        northward_wind_velocity         = raw_data[10]
-
-        global INSTRUMENT_PARTICLE_MAP
-        del INSTRUMENT_PARTICLE_MAP[:]      # Clear the list before making a new one
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(barometric_pressure, 'barometric_pressure', SENSOR_GROUP_BAROMETRIC_PRESSURE))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(relative_humidity, 'relative_humidity', SENSOR_GROUP_RELATIVE_HUMIDITY))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(air_temperature, 'air_temperature', SENSOR_GROUP_AIR_TEMPERATURE))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(longwave_irradiance, 'longwave_irradiance', SENSOR_GROUP_LONGWAVE_IRRADIANCE))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(precipitation, 'precipitation', SENSOR_GROUP_PRECIPITATION))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(sea_surface_temperature, 'sea_surface_temperature', SENSOR_GROUP_SEA_SURFACE_TEMPERATURE))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(sea_surface_conductivity, 'sea_surface_conductivity', SENSOR_GROUP_SEA_SURFACE_CONDUCTIVITY))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(shortwave_irradiance, 'shortwave_irradiance', SENSOR_GROUP_SHORTWAVE_IRRADIANCE))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(eastward_wind_velocity, 'eastward_wind_velocity', SENSOR_GROUP_EASTWARD_WIND_VELOCITY))
-        INSTRUMENT_PARTICLE_MAP.append(self.add_particle(northward_wind_velocity, 'northward_wind_velocity', SENSOR_GROUP_NORTHWARD_WIND_VELOCITY))
-
-    @staticmethod
-    def add_particle(raw_data, particle, group):
-        """
-        This function will add particle to the instrument particle map
-        """
-        if isinstance(raw_data, float):
-            return particle, group, float
-        else:
-            return particle, group, str
+                if not re.findall(r'\[.*\]:[a-z|A-Z]+', line):                  # Disregard anything that has a character after [Something]:
+                    line = re.sub(r'\[.*\]:', '', line)
+                    raw_data = line.split()
+                    raw_data[0:2] = [' '.join(raw_data[0:2])]                   # Merge the first and second elements to form a timestamp
+                    if raw_data is not None:
+                        for i in range(1, len(raw_data)):                       # Ignore 0th element, because that is the timestamp
+                            raw_data[i] = self.select_type(raw_data[i])
+                    # self.construct_instrument_particle_map(raw_data)
+                    particle = self._extract_sample(particle_class,
+                                                    None,
+                                                    raw_data,
```

Comment:
`raw_data` still has the timestamp in it at this point, I think you can do something like:
```python
time, date, raw_data = line.split()
raw_data[0] = raw_data[0].split(':')[-1]
``` 
And then you'll just have the raw_data to parse in `_extract_sample`.